UX-STANDARDS.md

```diff
@@ -0,0 +1,11 @@
+# UX Standards
+
+These standards include some basic user experience guidelines we align to on the marketing website, docs, and the registry.
+
+- Every page should have exactly one primary action.
+- Use descriptive text for button labels and links, and avoid labels such as "Learn more".
+- Use unique labels for distrinct actions.
```

Comment:
I'm sure I'd agree, but I'm not sure I necessary have the same thing in mind as you do here. An example or two here would help make this clearer as well.